apps/pm-user/src/components/atoms/input/Input.js

```diff
@@ -0,0 +1,34 @@
+import PropTypes from 'prop-types';
+import _noop from 'lodash/noop';
+import styles from './Input.module.scss';
+
+const Input = ({ name, onChangeHandler, placeholder, type, value }) => {
```

Comment:
input component should have one more prop to make field "required"

Reply:
Issue resolved.